File: leetcode/0268-missing-number.ts
function missingNumber(nums: number[]): number {
    let sum: number = 0;
    let arr: number = 0;

    for (let i = 0; i < nums.length; i++) {
        sum += i;
        arr += nums[i];
    }

    return sum + nums.length - arr;
}

File: leetcode/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum = 0, arr = 0;
        
        for(int i=0; i<nums.length; i++) {
            sum += i;
            arr += nums[i];
        }
        
        return sum + nums.length - arr;
    }
}